dist/NoSleep.js

```diff
@@ -100,6 +100,8 @@ var NoSleep = function () {
 
     _classCallCheck(this, NoSleep);
 
+    this.enabled = false;
```

Comment:
How about renaming this to `enableAutoRestart` or `enableAutoLoop` or `forceAutoRestart`?
If we want to merge these changes I believe we should make them sustainable

Reply:
It is not only meant for autorestart though - it simply indicates when the NoSleep is enabled or not.